src/OSPSuite.Core/Domain/Services/ModelValidator.cs

```diff
@@ -76,7 +66,11 @@ protected void CheckFormulaIn(IUsingFormula entity, IFormula formulaToCheck, Res
          var entityAbsolutePath = _objectPathFactory.CreateAbsoluteObjectPath(entity).ToPathString();
          var builder = _buildConfiguration.BuilderFor(entity);
          var objectWithError = builder ?? entity;
-         var entityType = _objectTypeResolver.TypeFor(entity);
+
+         // Dynamic formula may contain object path that will be resolved per instance. It cannot be checked here
+         if (formulaToCheck.IsDynamic())
```

Comment:
@Yuri05 This is the change here. Dynamic formula should bypass validation (the actual validation will happen when expanding dynamic formula)

So far we only had simple sum formula so we never had the issue :)